src/components/subscriptionDetail/SubscriptionInstance.tsx

```diff
@@ -18,12 +18,7 @@ import { isArray, partition } from "lodash";
 import React from "react";
 import { FormattedMessage } from "react-intl";
 
-interface ISubscriptionInstance {
-    subscription_instance_id: string;
-    name: string;
-    label?: string;
-    [index: string]: any;
-}
+import { ISubscriptionInstance } from "../../utils/types";
```

Comment:
you don't need the `../../`

Reply:
👍 I noticed I used relative paths in other imports as well. I cleaned those up as well.